index.html

```diff
@@ -869,7 +965,31 @@ <h3>Script Event</h3>
 
         <section>
           <h4>Text</h4>
-         <p>The <dfn>Text</dfn> object contains text content typically in a single language. This language may be the <a>Original</a> language or a <a>Translation</a> language.</p>
+          <p>The <dfn>Text</dfn> object contains text content typically in a single language.
+            This language may be the <a>Original</a> language or a <a>Translation</a> language.</p>
+          <p><a>Text</a> is defined as <dfn>Original</dfn> if it is any of:
+            <ol>
+              <li>the same language as the dialogue that it represents in the original programme audio;</li>
+              <li>a transcription of text visible in the programme video, in the same language as that text;</li>
+              <li>representative of non-dialogue sound;</li>
+              <li>a description of the scene in the programme video.</li>
+            </ol>
```

Comment:
The second two bullet points may need to be changed to

* **untranslated** description of non-dialogue sound
* an **untranslated** description of the scene in the programme video

Reply:
Done in 10b1a0e